android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboardDownloaderActivity.java

```diff
@@ -285,8 +286,7 @@ public static void downloadLexicalModel(Context context,
 
   public static boolean isCustom(String u) {
     boolean ret = false;
-    if (u != null && !u.contains(KMKeyboardDownloaderActivity.kKeymanApiBaseURL) &&
-      !u.contains(KMKeyboardDownloaderActivity.kKeymanApiRemoteURL)) {
+    if (u != null && !u.contains(kKeymanBaseURL)) {
```

Comment:
This seems like it loses the match on api.keyman.com.

Reply:
I thought we were getting away from api.keyman.com for downloading keyboards. But I can put that back

Reply:
I guess we are; I'd prefer to see that we treat `[*.]keyman.com` as non-custom for the future. This would mean testing `http(s?)://(.+\.)?keyman.com` or something like that I guess so we don't accidentally capture `https://amharickeyman.com` or things like that.

Reply:
I'll move this method to FileUtils and have it regex for [*.]keyman.com/

Gonna also remove 
`kKeymanApiRemoteURL = "https://r.keymanweb.com/api/2.0/remote?url=";`
since it doesn't seem to be used elsewhere.